### ISSUE TITLE
Sometimes we tell the user that a disruption is a minor disruption and all lines are blocked in the same text

### ISSUE THREAD

**Issue by mrmjprice** [wmcadigital/wmn-disruption-email-templates#30]
## Pain points
- [ ] User gets a mixed message, possibly due to incorrect labelling by West Midlands Trains
- [ ] Do we just tell the user there's a disruption for train disruptions, as a train disruption is always going to be some sort of delay?

## Example
```There is a Minor disruption Because of a fault with the signalling system between Longbridge and Redditch all lines towards Redditch are blocked.. Affects Cross City. Read more: https://disruptions.wmnetwork.co.uk/?selectedByMap=true&selectedItem=365512247```

**Comment by mrmjprice** [comment]
@somtmee @KaterinaKir @jkmnch 

What do you think about this?

**Comment by somtmee** [comment]
Yeah, it is a bit of a mixed message. All lines to the destination being blocked doesn't seem very minor. Do users classify the severity info as important? Do we have that insight from the Disruptions workshops?
